languages/en.ini

```
@@ -855,6 +865,7 @@ Message = "Message"
 Message From Sender = "Message From Sender"
 Metadata Prefix = "Metadata Prefix"
 Microfilm = "Microfilm"
+microform_viewing_request_text = "Microform materials can be accessed at the Main Library during %%link_start%%open hours%%link_end%%."
```

It looks like this is intended to contain HTML -- we use the _html suffix on strings that contain HTML as an indicator that they should not be HTML escaped. I think it would make sense to follow that convention here (and anywhere else that might be impacted).

```suggestion
microform_viewing_request_html = "Microform materials can be accessed at the Main Library during %%link_start%%open hours%%link_end%%."
```

---

The text itself is not supposed to contain html. The link is added (or not) in the code.
Let me know if you want me to add the suffix

---

True, but when the processing is complete, the resulting string contains HTML and should not be escaped, so I think the `_html` suffix is appropriate here to indicate that the string will not be escaped. This could be important if a user customizes/overrides it.

---

Done